dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/datafile/ingest/IngestReport.java

```diff
@@ -55,6 +60,12 @@ public void setId(Long id) {
     @Lob
     private String report;
 
+    private String argumentsBundleKey;
+
+    @ElementCollection
+    @OrderColumn
+    private List<String> reportArguments = new ArrayList<>();
```

Comment:
It seems redundant to save both the key and it's value from arbitrarily chosen language. How about only saving the bundle key and arguments under `report` & `reportArguments` respectively? Value of the `report` field could be the fallback in case of unsolvable key, but I'd enforce the bundle key here, as we want to enforce i18n.

Reply:
The problem is that we already have reports in database, so we can't put the bundle key in `report` field.

Reply:
Do we already have a production installation with actual production data that cannot be changed? Or do you mean the test instances only? I though we didn't have a public release yet, did we? Even if we did, current values would follow the fallback scenario and still work just fine. And if we have existing records, don't we want translations for them too as a database changeset that converts raw messages into bundle keys?

Reply:
We don't but during conversation with Krzysiek and Wojtek I received information that we have to think about reports that are present. I haven't asked why in this case. 

> Even if we did, current values would follow the fallback scenario and still work just fine.

But in the case of not finding a key in bundle if it is a key we would return useless key id to the user.

> And if we have existing records, don't we want translations for them too as a database changeset that converts raw messages into bundle keys?

 Well we were under the impression that converting messages to bundle keys is slippery slope. 

I think the best thing to do is talk about it during daily, I don't want to break the agreements that I made without them knowing, ok?

Reply:
Yep, let's talk it through with them tomorrow morning.